master/internal/checkpoint_gc.go

```diff
@@ -89,5 +92,20 @@ func (t *checkpointGCTask) completeTask(ctx *actor.Context) {
 	if err := t.db.CompleteTask(t.taskID, time.Now().UTC()); err != nil {
 		ctx.Log().WithError(err).Error("marking GC task complete")
 	}
+
+	var deleteCheckpoints []uuid.UUID
+	if err := json.Unmarshal(t.ToDelete, &deleteCheckpoints); err != nil {
+		ctx.Log().WithError(err).Error("unmarshalling ToDelete in checkpoint GC task")
+	}
+
+	var deleteCheckpoints_str []string
+	for _, dC := range deleteCheckpoints {
+		deleteCheckpoints_str = append(deleteCheckpoints_str, dC.String())
```

Comment:
@stoksc I'm converting to a string because a lot of the methods expect [] string type because it depends on the DeleteCheckpointsRequest in checkpoint.proto which has  repeated string checkpoint_uuids = 1.